assets/js/components/notifications/SetupSuccessBannerNotification.js

```diff
@@ -262,8 +261,8 @@ function SetupSuccessBannerNotification() {
 				</Fragment>
 			);
 
-		case 'user_input_success':
-			return <UserInputSuccessBannerNotification />;
+		default:
```

Comment:
Although I did actually [ask](https://github.com/google/site-kit-wp/issues/7429#issuecomment-1697795638) for the IB to specify removing the `switch` statement, upon further review I decided to leave it, as we are also checking for `notification === 'authentication_success'` elsewhere in this component, so it seemed appropriate to leave the check in the switch too. We can clean this up more at some point.

https://github.com/google/site-kit-wp/blob/761691772997c173a64ef0e2531121f45314cbda/assets/js/components/notifications/SetupSuccessBannerNotification.js#L169-L170